linode/manager
# hotfix: [M3-8182] - Remove leaveDelay from TooltipText

## Description 📝
A bug/artifact [I introduced](https://github.com/linode/manager/pull/10473/files#diff-12683df425c3158844e1a0ae2f06f393deb4a8d757ffddda5ea966624c52730dR69) for testing purposes while writing an end to end test (mea culpa 😭).

## Changes  🔄
- Remove erroneous `leaveDelay` on `<TextTooltip />`
- Add unit

![Screenshot 2024-05-29 at 10 54 08](https://github.com/linode/manager/assets/130582365/1ef291d9-7b75-4c35-893b-4bbdd44b328d)

## Target release date 🗓️
5/29/2024

## How to test 🧪
Use PROD

### Reproduction steps
- Go to linode/create
- select a region
- Scroll down the the details panel
- Hover over Placement Groups "region"
- Notice tooltip won't go away

### Verification steps
- Go to linode/create
- select a region
- Scroll down the the details panel
- Hover over Placement Groups "region"
- Confirm tooltip goes away

## As an Author I have considered 🤔

*Check all that apply*
